### PR TITLE
feat(sentry): add sentry

### PR DESCRIPTION
**Pull request by philibea**
## Type

- Feature

#### The following changes where made:

1. Add sentry  configuration template

2. Add Sentry Secret

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,3 +35,12 @@ jobs:
           node-version: 16
       - run: yarn --pure-lockfile
       - run: yarn run build
+        env:
```
Comment (chambo-e) [inline .github/workflows/ci.yml:38]: Should only be done on deploy no ?

File: .sentryclirc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[defaults]
+url=https://sentry.scaleway.com
```
Comment (chambo-e) [inline .sentryclirc:2]: This is already specified via secrets no ?

File: sentry.server.config.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const { init } = require('@sentry/nextjs')
```
Comment (chambo-e) [inline sentry.server.config.js:1]: We don't have a servers :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (philibea) [reply .github/workflows/ci.yml:38]: This will break build too in this case. We should make only one build in our ci

File: .sentryclirc (hunk 2 above)
Comment (philibea) [reply .sentryclirc]: Remove

**Review by chambo-e — APPROVED** [review]

**Pull request merged**